src/inc/winwrap.h

```diff
@@ -48,6 +47,7 @@
 #include <specstrings.h>
 
 #include "registrywrapper.h"
+#include "winwrappers.h"
```

Comment:
I would call this "longfilepathwrappers.h" - winwrappers.h is too generic and does not communicate the intent.